src/components/AddComment.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import { postComment } from "../utils/API";
+
+const AddComment = ({ username, comments, setComments }) => {
+  const [newComment, setnewComment] = useState("");
+  const { review_id } = useParams();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    postComment(review_id, "weegembump", newComment)
+      .then((addedComment) => {
+        setComments((currComments) => {
+          return [addedComment, ...currComments];
+        });
+        setnewComment("");
+        alert("comment added succesfully!");
+      })
+      .catch((err) => {
+        alert("You must sign in to comment!");
```

Comment:
There could potentially have errors other than not being signed in - where/when you handle the error is sound - consider an alternative message for the user